### PR TITLE
Acceleration docs

### PR DESCRIPTION
**Pull request by mitchdevenport**

### CODE REVIEW

**Review by ewgenius — DISMISSED** [review]

File: spiceaidocs/content/en/reference/Spicepod/datasets.md [hunk 1]
```diff
@@ -108,13 +108,35 @@ Optional. Accelerate queries to the dataset by caching data locally.
...
+Enable or disable acceleration, defaults to `true`.
 
-## `acceleration.refresh`
+## `acceleration.engine`
 
-Optional. The interval to refresh the data for the dataset if the dataset type is overwrite.
+The acceleration engine to use, defaults to `arrow`. The following engines are supported:
 
-For `append` datasets, the refresh interval not used.
+  - `arrow` - Cache data as in-memory Apache Arrow Records.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Spicepod/datasets.md]: Remove the word Cache.

Use "Accelerated in-memory backed by Apache Arrow DataTables`

File: spiceaidocs/content/en/reference/Spicepod/datasets.md [hunk 2]
```diff
@@ -108,13 +108,35 @@ Optional. Accelerate queries to the dataset by caching data locally.
...
+  - `duckdb` - Cache data in a DuckDB.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Spicepod/datasets.md]: "Accelerated by an embedded DuckDB database" (with link to DuckDB)

File: spiceaidocs/content/en/reference/Spicepod/datasets.md [hunk 3]
```diff
@@ -108,13 +108,35 @@ Optional. Accelerate queries to the dataset by caching data locally.
...
+  - `postgres` - Cache data in a PostgreSQL database.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Spicepod/datasets.md]: "Accelerated by a PostgreSQL database"

File: spiceaidocs/content/en/reference/Spicepod/datasets.md [hunk 4]
```diff
@@ -108,13 +108,35 @@ Optional. Accelerate queries to the dataset by caching data locally.
...
+
+## `acceleration.mode`
+
+Optional. The mode of acceleration. The following values are supported:
+
+  - `memory` - Cache data in memory.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Spicepod/datasets.md]:
```suggestion
  - `memory` - Store acceleration data in-memory.
```

File: spiceaidocs/content/en/reference/Spicepod/datasets.md [hunk 5]
```diff
@@ -108,13 +108,35 @@ Optional. Accelerate queries to the dataset by caching data locally.
...
+  - `file` - Cache data in a file.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Spicepod/datasets.md]:
```suggestion
  - `file` - Store acceleration data in a file.
```

File: spiceaidocs/content/en/reference/Spicepod/datasets.md [hunk 6]
```diff
@@ -108,13 +108,35 @@ Optional. Accelerate queries to the dataset by caching data locally.
...
+
+`mode` is only supported for the `duckdb` engine.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Spicepod/datasets.md]:
```suggestion
`mode` is currently only supported for the `duckdb` engine.
```

File: spiceaidocs/content/en/reference/Spicepod/datasets.md [hunk 7]
```diff
@@ -123,3 +145,11 @@ i.e. `1h` for 1 hour, `1m` for 1 minute, `1s` for 1 second, etc.
...
 
 If not specified, the default retention is to keep all data.
+
+## `acceleration.params`
+
+Optional. Parameters to pass to the acceleration engine. The parameters are specific to the acceleration engine used.
+
+## `acceleration.engine_secret`
+
+Optional. The secret to use to connect to the acceleration engine. Use `spice login` to create a new secret.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Spicepod/datasets.md]:
```suggestion
Optional. The secret store key to use the acceleration engine connection credential. For supported data connectors, use `spice login` to store the secret.
```

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**